components/eamxx/src/share/grid/remap/coarsening_remapper.cpp

```diff
@@ -68,6 +68,21 @@ CoarseningRemapper (const grid_ptr_type& src_grid,
   scorpio::grid_read_data_array(map_file,"S",  -1,S_h.data(),       nlweights);
   scorpio::eam_pio_closefile(map_file);
 
+  // Offset the cols ids to match the source grid.
+  // Determine the min id among the cols array, we
+  // also add the min_dof for the grid.
+  int remap_min_dof = col_gids_h[0];
```

Comment:
This might be an issue if we have a rank which reads no triplets, since `col_gids_h` will have size 0. I think you should init `remap_min_dof` with `std::numeric_limits<int>::max()`.